CHANGELOG.md

```diff
@@ -6,6 +6,89 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 
+## [3.0.0] - 2019-03-27
+
+This release comes with a modernized ``ES6``-class structured code base as well
+as some significant local refactoring work regarding how ``Stack`` and ``Memory``
+are organized within the VM execution engine. It also finalizes the module-structuring
```

Comment:
> It also finalizes the module-structuring

Hopefully this doesn't mean the door for future module-structuring is closed after this release? :smile:

Reply:
Have toned down a bit. 😛